requirements.txt

```diff
@@ -0,0 +1,2 @@
+git+https://github.com/fact-project/fact_conditions#egg=fact_conditions
+git+https://github.com/mackaiver/gridmap#egg=gridmap
```

Comment:
Please use the form `https://github.com/fact-project/archive/<version>.tar.gz`, if there is not version, use `master` or make a release.

Installation is faster and we use a fixed version.

Reply:
fact_conditions got a version

gridmap does nto have a tag or version so I gave it the commit hash so it is a fixed version